doc/classes/AudioStreamSample.xml

```diff
@@ -110,20 +112,45 @@
 	</methods>
 	<members>
 		<member name="data" type="PoolByteArray" setter="set_data" getter="get_data">
+			Raw audio data.
 		</member>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="AudioStreamSample.Format">
+			Sets the audio format to [FORMAT_8_BITS], [FORMAT_16_BITS], [FORMAT_IMA_ADPCM].
```

Comment:
Don't use "sets" - the member variable is a setter and a getter.

Audio format. See FORMAT_* constants for values.